package.json

```diff
@@ -86,7 +86,7 @@
     "serve": "^11.3.0",
     "start-server-and-test": "^1.11.0",
     "styled-components": "^4.2.0",
-    "typescript": "^3.8.3",
+    "typescript": "4.1.2",
```

Comment:
My only concern dima, is that we will use a different version as Uniswap for getting this fancy feature. I would be worried if this breaks anything in the future or adds a the need to "hack" pieces of their code to migrate to this version of TS.